src/pages/carbon-offsetting.jsx

```diff
@@ -0,0 +1,564 @@
+          </p>
+
+          <p>
+            For example, <a href="https://offset.earth/">Offset Earth</a> do
+            support reforestation projects such as{' '}
+            <a href="https://edenprojects.org/">Eden Reforestation</a>, but they
+            also support Gold Standard certified projects like these:
+          </p>
+
+          <ul>
+            <li>
+              <a href="https://offset.earth/projects/istanbul-landfill-gas-extraction">
+                Istanbul Landfill Gas Extraction
+              </a>
+            </li>
+            <li>
+              <a href="https://offset.earth/projects/peruvian-protection-of-the-amazon">
+                Peruvian Protection of the Amazon
+              </a>
+            </li>
+            <li>
+              <a href="https://offset.earth/projects/solar-project-in-maharashtra-india">
+                Solar power in Maharashtra, India
+              </a>
+            </li>
+            <li>
+              <a href="https://offset.earth/projects/bio-digesters-for-rural-vietnamese-farmers">
+                Bio-digesters for rural Vietnamese farmers
+              </a>
+            </li>
+            <li>
+              <a href="https://offset.earth/projects/improved-stoves-in-eritrea">
+                Improved stoves in Eritrea
+              </a>
+            </li>
+            <li>
+              <em>
+                <a href="https://offset.earth/projects/">
+                  Loads of other projects
+                </a>
+              </em>
+            </li>
+          </ul>
+
+          <p>
+            These are all amazing projects picked from the{' '}
+            <a href="https://registry.goldstandard.org/projects">
+              Gold Standard Registry
+            </a>{' '}
+            which need more support to grow their impact and improve quality of
+            life for people all over the developing world.
+          </p>
+
+          <p>
+            Another interesting approach is directly capturing carbon dioxide
+            from the air is another interesting approach which seems to be
+            slowly improving over time, with
+            <a href="">Climeworks</a> https://climeworks.shop/) offering a
+            monthly subscription to back their project, which is about €50 a
+            month to remove 0.6 tonnes from the environment, which over could
+            wipe out 7.2 tonnes for €800. Are you willing to pay €800 a month to
+            offset most of your footprint?
+          </p>
+
+          <p>
+            Sending money to projects like this is solving problems around the
+            world, reducing and trapping emissions, and hopefully if they all
+            scale enough through any means then we'll see emissions drop, but
+            most people cannot afford to offset the amount they actually
+            produce: because we all produce too much CO2e and offsetting it
+            fully and legitimately is expensive.{' '}
```

Comment:
Run on sentence.  Fix:
"Sending money to projects like this is reducing and trapping emissions around the world. Hopefully, if these projects scale enough then we'll see emissions drop.  Unfortunately, most people cannot afford to offset the amount they actually produce because we all produce too much CO2e and offsetting it is fully and legitimately is expensive."